cluster/images/hyperkube/Makefile

```diff
@@ -60,43 +60,15 @@ ifndef VERSION
     $(error VERSION is undefined)
 endif
 	cp -r ./* ${TEMP_DIR}
-	mkdir -p ${TEMP_DIR}/cni-bin ${TEMP_DIR}/addons ${TEMP_DIR}/addons/singlenode ${TEMP_DIR}/addons/multinode
```

Comment:
Looks like there are other scripts relying on this `cni-bin` dir. Was removing this intentional?
It seems to have broken federation build.